### ISSUE TITLE
Fine tuning LSTM_rezoom  for  new image set

### ISSUE THREAD

**Issue by lewjae** [Russell91/TensorBox#105]
Hi,
I have run TensorBox (wth LSTM_rezoom) on my own image set (439x409x3) to detect buildings. I got some reasonable results, but the RP curve from "evaluate.ipynb" seemed to be less desirable. I am wondering if I could improve it by fine tuning the model.

![screen shot 2016-10-31 at 4 44 19 pm](https://cloud.githubusercontent.com/assets/2567400/20032766/d9ab65c2-a34e-11e6-9636-ee5bdde7e60c.png)

![screen shot 2016-11-05 at 11 11 43 am](https://cloud.githubusercontent.com/assets/2567400/20032514/f2eb20a0-a348-11e6-8d0d-68e3e6095d7f.png)

![screen shot 2016-11-05 at 11 28 08 am](https://cloud.githubusercontent.com/assets/2567400/20032610/fa273c4e-a34a-11e6-8069-196f9ce31ea2.png)

I also tried different learning rates.  "Loss" and "Accuracy" changed a little in TensorBoard but the RP curve remained same.  I am open for any suggestions. Here is my lstm_rezoom.json: 
    
    _"solver": {
        "opt": "RMS",
        "use_jitter": false,
        "rnd_seed": 1,
        "epsilon": 0.00001, 
        "learning_rate": 0.0002, 
        "learning_rate_step": 18000, 
        "hungarian_iou": 0.25,
        "weights": "",
        "head_weights": [1.0, 0.1]
    },
    "use_lstm": true,
    "use_rezoom": true,
    "biggest_box_px": 10000,
    "rezoom_change_loss": "center",
    "rezoom_w_coords": [-0.25, 0.25],
    "rezoom_h_coords": [-0.25, 0.25],
    "reregress": true,
    "focus_size": 1.8,
    "early_feat_channels": 256,
    "avg_pool_size": 5,
    "num_lstm_layers": 2,
    "image_width": 416, 
    "image_height": 384,
    "grid_height": 12, 
    "grid_width": 13,
    "batch_size": 1, 
    "region_size": 32,
    "clip_norm": 1.0,
    "lstm_size": 500,
    "deconv": false,
    "num_classes": 2,
    "rnn_len": 5_

One note -  I have to assign  "image_width": 416 and  "image_high":384 to be meet the "region _size":32 requirement (issue #30) even though the original image size is 439x406.  Does this have any impact on the RC curve result?  Thanks in advance.

**Comment by Russell91** [comment]
The evaluation criterion of 0.5 IntersectionOverUnion is actually quite strict. You could try reducing that requirement.
